CHANGELOG.md

```diff
@@ -1,3 +1,98 @@
+6.0.0 (16/01/2023)
+
+Supported API versions:
+* v3: [pa,spar,vault,pub,fi,axp,fpo,afi,npo,bpm,quant]
+
+Summmary:
+* Supporting new functionalities in pa,spar,vault,pub,quant and fi.
+
+Functionality Additions:
+* New Parameters are added in the FI Request.
+* 413 status code is added to Quant Reponse.
+* 'Warnings' are added as an enhancement to quant unit status response object.
+* In the Error response object new fields - code and title are added.
+* Added new end point for SPAR Product to get component details by Id.
+
+Breaking changes:
+* Removed defaults values when the parameter field is required for look up end points for pa,spar,vault and strategy end points.
```

Comment:
Removed default values when the parameter field is `required` for lookup end points for pa,spar,vault and strategy document end points for optimizers.